test/IISIntegration.FunctionalTests/HttpsTest.cs

```diff
@@ -138,7 +138,13 @@ public Task Https_HelloWorld_ClientCert_CoreCLR_X64_Portable()
                     ServerConfigTemplateContent = (serverType == ServerType.IISExpress) ? File.ReadAllText("Https.config") : null,
                     SiteName = "HttpsTestSite", // This is configured in the Https.config
                     TargetFramework = runtimeFlavor == RuntimeFlavor.Clr ? "net461" : "netcoreapp2.0",
-                    ApplicationType = applicationType
+                    ApplicationType = applicationType,
+                    Configuration =
+#if DEBUG
```

Comment:
where is DEBUG defined? VS by default only defines _DEBUG

Reply:
I believe it is defined by MSBuild. Other tests of ours use the same check, I just forgot to add it here too.

Reply:
See: https://github.com/aspnet/IISIntegration/pull/500/files/ba6658cc25d7d55c2feab7bf69005dab9c771bfb#diff-ba13b4c289f087409caa63ff7ae00bd1R60